infrastructure/src/main/java/org/corfudb/infrastructure/log/StreamLogFiles.java

```diff
@@ -1186,6 +1197,22 @@ public LogData read(long address) {
         }
     }
 
+    @Override
```

Comment:
I think we need JavaDoc here.

Reply:
Also maybe we can give a more clear name to the method.

Reply:
The JavaDoc is provided in the public interface.